src/mesh/eth/ethClient.cpp

```diff
@@ -66,8 +66,9 @@ static int32_t reconnectETH()
                 syslog.enable();
             }
 
-            // initWebServer();
+#if !MESHTASTIC_EXCLUDE_SOCKETAPI
```

Comment:
This is also a change that is just master dependent. Can we push this?

Reply:
What about devices that run a web server on Ethernet right now?

Reply:
@caveman99